csharp/src/Ice/OutgoingResponseFrame.cs

```diff
@@ -186,11 +199,15 @@ public static OutgoingResponseFrame WithVoidReturnValue(Current current)
                         }
                         else
                         {
+                            // We are forwarding an ice1 system exception using ice2.  We create an ice2 encapsulation
```

Comment:
For consistency with the previous comments, I would use: We are forwarding a system exception encoded using 1.1 and received over ice1 to ice2.